src/python/pants/backend/python/framework/stevedore/target_types.py

```diff
@@ -11,13 +11,13 @@
 
 
 class StevedoreNamespace(str):
-    f"""Tag a namespace in entry_points as a stevedore namespace.
+    """Tag a namespace in entry_points as a stevedore namespace.
 
     This is required for the entry_point to be visible to dep inference
     based on the `stevedore_namespaces` field.
 
     For example:
-        {PythonDistribution.alias}(
+        python_distribution(
```

Comment:
Just looking now, it looks like this code block isn't formatted as intended in the online rendering, as it ends up as just plain text:

<img width="608" alt="image" src="https://github.com/pantsbuild/pants/assets/1203825/080a917c-1472-4c65-b3af-540ad2b08197">

I think we could either:

- add an extra newline
  ```suggestion
  
          python_distribution(
  ```
- wrap this in backticks and decent
  ````suggestion
      ```python
      python_distribution(
  ````


The second one gives better syntax highlighting.

<img width="453" alt="image" src="https://github.com/pantsbuild/pants/assets/1203825/5ac47910-1ad8-488c-9977-2d0902d2351c">

Reply:
I'll trade the nicer rendered docs over a more readable CLI help as being the more exposed one.